fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -0,0 +1 @@
+client for the free software media server
```

Comment:
```suggestion
Mobile app for Jellyfin, the free software media system
```

Reply:
If you insist :wink: Usually we leave the obvious out of the summary. To give an extreme example: "open source app of X for android" => F-Droid only caters open source apps for Android. So "of X" would be the only thing of substance :smile: Compromise: "Mobile client for Jellyfin, the free software media system"?

Reply:
> If you insist :wink: Usually we leave the obvious out of the summary. To give an extreme example: "open source app of X for android" => F-Droid only caters open source apps for Android. So "of X" would be the only thing of substance :smile: Compromise: "Mobile client for Jellyfin, the free software media system"?

Sounds good, but I'd capitalize the first character (so, "Mobile" instead of "mobile").

Reply:
The important part for me was to include the "mobile" part as we have another Android app for televisions

Reply:
I've left it lower-case as it's not starting a sentence – but again… :wink: will do.